Improve READMEs (#2708)
theforeman/foreman-documentation: guides/README.md

* [attributes-satellite.adoc](common/attributes-satellite.adoc): base overrides for satellite build.
* [attributes-orcharhino.adoc](common/attributes-orcharhino.adoc): base overrides for orcharhino build.

By default, attributes cannot be used in shell or code examples.
To use them, use the "attributes" keyword:

	[options="nowrap" subs="+quotes,attributes"]
	----
	# ls {AttributeName}
	----

Avoid using phrases like "Starting from version 6.5 or 1.22" because it is not possible to easily translate these strings into all build variants.

#### Conditional Content

If a piece of your content, such as a specific block, paragraph, warning or chapter, is specific for a certain [build](#builds), use special build attributes to show or hide it.

To show a piece of content only for the `katello` build:

	ifdef::katello[]
	NOTE: This part is only relevant for Foreman with the Katello plugin.
	endif::[]

> To hide a piece of content, i.e. it will be shown for all other builds:

Reviewer: ```suggestion
To hide a piece of content means that it will be shown for all other builds:
```

Unsure about the wording here :shrug:

* [`snip`](https://redhat-documentation.github.io/modular-docs/#using_text_snippets_or_text_fragments_writing-mod-docs): Files starting with `snip_` contain snippets that are reused throughout multiple guides, e.g. admonitions.
Snippets do not require an ID.

### Creating new guides

> Each guide must be in a separate directory in guides/ prefixed with "doc-". The following requirements must be met:

Reviewer: ```suggestion
Each guide must be in a separate directory in guides/ prefixed with "doc-".
All guides must meet the following requirements:
```

* Top-level file `master.adoc`.

> * Directory `topics` with actual content.

Reviewer: ```suggestion
```

This is no longer a requirement. For example, all content for "Managing Hosts" is keep within "guides/common/modules/".

* Symlink `common` to `../common` for shared content.
* Directory `images` with images.
* Symlink `images/common` to `../common/images`.
* File `Makefile` which includes `../common/Makefile`.

> * File `docinfo.xml` for downstream (Red Hat) guides. Presence of this file will cause build check via `ccutil`, a tool used by Red Hat documentation team.

Reviewer: ```suggestion
* File `docinfo.xml` for Red Hat Satellite guides.
Presence of this file will cause build check via `ccutil`, a tool used by Red Hat documentation team.
```

* Put new content into the `guides/common/modules` directory and stick to one (level one) heading per file.
* Update `web/content/js/nav.js` to add a new item on the top-level menu.

New guides can be typically created as copies of existing guides, just make sure to clean images, topics and `docinfo.xml`.

### Reviewing

> If the review would result in many comments in a GitHub pull request, consider checking out the topic branch and putting necessary changes on top of author's work instead of making the comments.

Reviewer: I am generally OK with this, but don't practice it myself. Let's talk about this next meeting (not blocking this PR!)

Reviewer: https://github.com/theforeman/foreman-documentation/pull/2742
